src/View/ViewableData.php

```diff
@@ -293,7 +293,7 @@ public function castingHelper($field)
         }
 
         // Fall back to default_cast
-        $default = $this->config()->get('default_cast');
+        $default = self::config()->get('default_cast');
```

Comment:
I'm with @tractorcow, make it consistent throughout the whole with `static::config` or `ClassName::config()` where possible without breaking.